optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -263,6 +274,28 @@ export const TrialTable: FC<{
   ]
 
   const collapseBody = (index: number) => {
+    const objectiveFormRefs = studyDetail?.directions.map((d) =>
+      createRef<HTMLInputElement>()
+    )
+    const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
+      if (objectiveFormRefs === undefined) {
+        return
+      }
+
+      e.preventDefault()
+      const studyId = (studyDetail as StudyDetail).id
+      const trialId = trials[index].number
+      const objectiveValues = objectiveFormRefs.map((ref) =>
+        ref.current ? Number(ref.current.value) : NaN
```

Comment:
I think we should not call APIs when `ref.current` is undefined or null. It's an implicit behavior that may cause the critical bug. What do you think?

Reply:
Thank you. I changed it as suggedsted.